src/sinks/websocket.rs

```diff
@@ -0,0 +1,609 @@
+use crate::{
```

Comment:
It would be nice if this file was broken up into multiple files. (`WebSocketSink` in 1 file, `WebSocketSinkConfig` in another, etc)